great_expectations/types/__init__.py

```diff
@@ -236,12 +238,12 @@ def assert_valid_keys(keys: Set[str], purpose: str) -> None:
 
 
 class SerializableDictDot(DictDot):
-    def to_json_dict(self) -> dict:
-        """
+    def to_json_dict(self) -> Dict[str, JSONValues]:
```

Comment:
This is a method that needs to be implemented by subclasses. I didn't know if we considered that as public (it's not the spreadsheet) so I didn't add the decorator but I still wanted to add the docstring.